### ISSUE TITLE
Error: [libs\datamodel\core\src\dml\datamodel.rs:166:9] Every RelationInfo should have a complementary RelationInfo on the opposite relation field. 

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#3317]
Command: `prisma introspect`
Version: `2.5.0-dev.61`
Binary Version: `cc6843945c4485b397f541fcbe18d1bd94346ef7`
Report: https://prisma-errors.netlify.app/report/10277 
OS: `x64 win32 10.0.18362`

JS Stacktrace:
```
Error: [libs\datamodel\core\src\dml\datamodel.rs:166:9] Every RelationInfo should have a complementary RelationInfo on the opposite relation field.
    at IntrospectionEngine.<anonymous> (C:\Users\Jan\Documents\throwaway\prisma-client-js-benchmarks\node_modules\@prisma\cli\build\index.js:2:1763646)
    at Generator.throw (<anonymous>)
    at rejected (C:\Users\Jan\Documents\throwaway\prisma-client-js-benchmarks\node_modules\@prisma\cli\build\index.js:2:1760649)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
```

Rust Stacktrace:
```
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: <unknown>
  19: <unknown>
  20: <unknown>
  21: <unknown>
  22: <unknown>
  23: <unknown>
  24: <unknown>
  25: <unknown>
  26: <unknown>
  27: <unknown>
  28: <unknown>
  29: <unknown>
  30: <unknown>
  31: <unknown>
  32: <unknown>
  33: <unknown>
  34: BaseThreadInitThunk
  35: RtlUserThreadStart

```

This is the schema:
```
// datasource db {
//   provider = "postgres"
//   url      = "postgres://postgres:prisma@localhost:5432/postgres?schema=sql_load_test"
// }

datasource db {
  provider = "sqlite"
  url      = "file:benchmarks.db"
}

generator client {
  provider = "prisma-client-js"
}

model Comment {
  author    Int
  content   String?
  createdAt DateTime @default(dbgenerated())
  id        Int      @id
  post      Int
  updatedAt DateTime @default(dbgenerated())
  User      User     @relation(fields: [author], references: [id])
  Post      Post     @relation(fields: [post], references: [id])
  Like      Like[]

  @@index([author], name: "Comment.author")
  @@index([post], name: "Comment.post")
}

model Like {
  comment   Int
  createdAt DateTime @default(dbgenerated())
  id        Int      @id
  post      Int
  updatedAt DateTime @default(dbgenerated())
  user      Int
  Comment   Comment  @relation(fields: [comment], references: [id])
  Post      Post     @relation(fields: [post], references: [id])
  User      User     @relation(fields: [user], references: [id])

  @@index([comment], name: "Like.comment")
  @@index([post], name: "Like.post")
  @@index([user], name: "Like.user")
}

model Post {
  author    Int
  content   String?
  createdAt DateTime  @default(dbgenerated())
  id        Int       @id
  updatedAt DateTime  @default(dbgenerated())
  User      User      @relation(fields: [author], references: [id])
  Comment   Comment[]
  Like      Like[]

  @@index([author], name: "Post.author")
}

model User {
  age                            Int?
  createdAt                      DateTime     @default(dbgenerated())
  email                          String?
  firstName                      String       @default("")
  id                             Int          @id
  lastName                       String       @default("")
  password                       String?
  updatedAt                      DateTime     @default(dbgenerated())
  Comment                        Comment[]
  Like                           Like[]
  Post                           Post[]
  FriendShip_UserTo_FriendShip_A FriendShip[] @relation("UserTo_FriendShip_A")
  FriendShip_UserTo_FriendShip_B FriendShip[] @relation("UserTo_FriendShip_B")
}

model FriendShip {
  A                        Int
  B                        Int
  User_UserTo_FriendShip_A User @relation("UserTo_FriendShip_A", fields: [A], references: [id])
  User_UserTo_FriendShip_B User @relation("UserTo_FriendShip_B", fields: [B], references: [id])

  @@map("_FriendShip")
  @@unique([A, B], name: "_FriendShip_AB_unique")
}

```

Database is available for download at https://github.com/prisma/prisma-client-js-benchmarks/blob/eecf7499ac63d4c8fce1a7064fc0492a2df1fdb9/prisma/benchmarks.db

**Comment by tomhoule** [comment]
This should be fixed by https://github.com/prisma/prisma-engines/pull/1033 (will be merged soon)

**Comment by pantharshit00** [comment]
I can confirm this has been fixed in `2.0.0-dev.62`.

This also didn't exist in the latest stable which is 2.4.1 so this was a dev build regression which is fixed now so I am going to close this.

![image](https://user-images.githubusercontent.com/22195362/90395671-847ff400-e0b2-11ea-9412-cd4bb6b93a49.png)
